### PR TITLE
Mahesh Assignment_24_25Mins_Done

### PR DESCRIPTION
**Pull request by MaheshKJun20**
image
Mahesh_Assignment_24_25Mins_Done

### CODE REVIEW

File: src/maheshKJun20/NumbersExample/ValidatePhoneNumber.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+				break;
+			default:
+				System.out.println(phoneNumber + " - is invalid phone number");
+		}
+	}
+	
+	//Method to validate the country code is valid or not
+	boolean isValidCountryCode(String countryCode) {
+		boolean flag = true;
+		if (!String.valueOf(countryCode.charAt(0)).equals("+") || !String.valueOf(countryCode.charAt(1)).equals("9") || !String.valueOf(countryCode.charAt(2)).equals("1") || !String.valueOf(countryCode.charAt(3)).equals(" ")) {
```
Comment (harshvegada) [inline src/maheshKJun20/NumbersExample/ValidatePhoneNumber.java]: here condition should be && not ||

**Review by harshvegada — APPROVED** [review]
Code logic is correct @MaheshKJun20 
Keep it up!
one comment check n update code

File: src/maheshKJun20/NumbersExample/ValidatePhoneNumber.java (hunk 1 above)
Comment (MaheshKJun20) [reply src/maheshKJun20/NumbersExample/ValidatePhoneNumber.java]: Thanks Harsh. Updated The code as per the comments.

**Review by harshvegada — APPROVED** [review]
Looks correct code!

**Pull request merged**